docs/src/misc.rst

```diff
@@ -541,6 +541,10 @@ API
     memory allocated to `pwd` needs to be freed with
     :c:func:`uv_os_free_passwd`.
 
+    .. note::
+        If the effective uid does not have an entry in the password file this
+        function will return ENOENT even if the file exists.
```

Comment:
```suggestion
        function will return `UV_ENOENT` even if the file exists.
```
I kind of feel we shouldn't be using the phrase "password file" since there doesn't have to be one (e.g., on systems that use LDAP-based authentication) but I'm not sure what to replace "password file entry" on line 536 with.

Reply:
I think "user source" or "user database" would fit well

Reply:
Yes, that's a good suggestion.

Reply:
I took a look at the windows source, and it looks like the specific ENOENT issue is only relevant for unix. Also, the term `password file` is used all over the documentation of this function.

Maybe prefixing this block with `On Unix systems, if the effective uid...` would be the way to go

Reply:
I'd consider UV_ENOENT being Unix-only an implementation detail. It's something that can change in the future. I wouldn't mention that.

Reply:
What can be particularly confusing here is that the authoritative information for linux systems now comes not directly from `/etc/password` but from `/etc/nsswitch.conf` which specifies a list of dynamic library (such as libnss_files.so) to open to search for pwnam info corresponding to the given uid. The man page has this to say about the error:

```
       0 or ENOENT or ESRCH or EBADF or EPERM or ...
              The given name or uid was not found.
```